content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -137,6 +138,14 @@ You can report issues with using or deploying this third-party device plugin by
 
 Google publishes its own [instructions](https://cloud.google.com/kubernetes-engine/docs/how-to/gpus) for using NVIDIA GPUs on GKE .
 
+### Deploying Intel GPU device plugin
+
+The Intel device plugins for Kubernetes repository hosts the
+[documentation for the Intel GPU device plugin](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/cmd/gpu_plugin/README.md),
+including the deployment instructions.
+
+The Intel GPU device plugin can be configured to enable GPU sharing and fractionalization.
```

Comment:
This section should appear _before_ the NVIDIA section.

Reply:
done